src/main/scala/com/ibm/couchdb/api/Documents.scala

```diff
@@ -47,16 +47,33 @@ class Documents(client: Client, db: String, typeMapping: TypeMapping) {
         CouchDoc[T](obj, types(cl)))
   }
 
+  private def updateBulk[T: upickle.Writer](objs: Seq[CouchDoc[T]]): Task[Seq[Res.DocOk]] = {
+    client.post[Req.Docs[T], Seq[Res.DocOk]](
+      s"/$db/_bulk_docs",
+      Status.Created, Req.Docs(objs))
```

Comment:
`Status.Created` is only relevant for `createMany`, should be set appropriately for delete and update queries.